### PR TITLE
General structure

### PR DESCRIPTION
**Pull request by marko-knoebl**
Implement general structure of the project

### CODE REVIEW

File: todolist.js [hunk 1]
```diff
@@ -1 +1,46 @@
...
+  while (choice !== "q") {
+    //displayTodos();
+    console.log(
+      "What do you want to do?\n" +
+        "[a] add a todo\n" +
+        "[d] delete a todo\n" +
+        "[c] mark a todo as completed\n" +
+        "[q] quit"
+    );
+    choice = prompt();
```
Comment (marko-knoebl) [inline todolist.js]: better variable name could be "command"

Comment (Kenavm) [reply todolist.js]: all's good

Comment (kchristoph30) [reply todolist.js]: nice code

Comment (MelinaPr) [reply todolist.js]: i like choice better

File: todolist.js [hunk 2]
```diff
@@ -1 +1,46 @@
...
+    if (choice === "a") {
```
Comment (TheresaGri) [inline todolist.js]: can you see this?

File: todolist.js (hunk 1 above)
Comment (Kata2306) [reply todolist.js]: 10/10

**Pull request merged**